code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -195,9 +198,38 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	cell_type = /obj/item/stock_parts/cell/gun/mini
 	ammo_type = list(/obj/item/ammo_casing/energy/lasergun/hitscan)
+	manufacturer = MANUFACTURER_SHARPLITE_NEW
 
 /obj/item/gun/energy/laser/hitscanpistol/examine_more(mob/user)
 	if(in_range(src, user) || isobserver(user))
-		. = list("<span class='notice'>You examine [src] closer. Under the grip is a small inscription: \"NT CN SVALINN 2502\".</span>")
+		. = list("<span class='notice'>You examine [src] closer. Under the grip is a small inscription: \"NT CN SVALINN 462\".</span>")
 	else
 		. = list("<span class='warning'>You try to examine [src] closer, but you're too far away.</span>")
+
+/obj/item/gun/energy/laser/e10
+	name = "E-10 laser pistol"
+	desc = "A very old laser weapon. Despite the extreme age of some of these weapons, they are sometimes prefered to the mass produced Nanotrasen laser weapons."
+	icon_state = "e10"
+	w_class = WEIGHT_CLASS_SMALL
+	ammo_type = list(/obj/item/ammo_casing/energy/lasergun/eoehoma)
+	manufacturer = MANUFACTURER_EOEHOMA
+
+/obj/item/gun/energy/laser/e50
+	name = "E-50 energy emitter"
+	desc = "A heavy and extremely powerful laser. Sets fires and kick ass, however uses a massive amount of energy per shot and is generally awkard to fire with."
```

Comment:
Grammar, wording, and spelling edits. This _does_ start fires right?

"A heavy and extremely powerful laser. Sets targets on fire and kicks ass, but it uses a massive amount of energy per shot and is generally awkward to handle."